app.js

```diff
@@ -0,0 +1,39 @@
+const cors = require("cors");
+let dotEnv = require("dotenv");
+const express = require("express");
+const mongoose = require("mongoose");
+const PORT = process.env.PORT || 5000;
```

Comment:
Put this after all the require statements.

Reply:
what, PORT?